scripts/sanity_chk/sanitylib.py

```diff
@@ -737,9 +746,22 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn, results, harness
 
         line = ""
         timeout_extended = False
+
+        pid = 0
+        cpu_time = 0
+        if os.path.exists(pid_fn):
+            pid = int(open(pid_fn).read())
+            cpu_time = QEMUHandler._get_cpu_time(pid)
+
         while True:
             this_timeout = int((timeout_time - time.time()) * 1000)
             if this_timeout < 0 or not p.poll(this_timeout):
+                if pid and this_timeout > 0:
```

Comment:
Add comment about why this is needed here please.

Reply:
@nashif update